src/components/BooksForm.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+const CATEGORIES = [
+  'Action',
+  'Biography',
+  'History',
+  'Horror',
+  'Kids',
+  'Learning',
+  'Sci-Fi',
+];
+
+function createCategories() {
+  return (
+    <div>
+      <label htmlFor="categories" />
+          Category:
+      <select id="categories">
+        { CATEGORIES.map((cat, idx) => <option key={idx} value={cat}>{cat}</option>) }
+      </select>
+    </div>
+  );
+}
+
+function BooksForm() {
+  return (
+    <form>
+      <label htmlFor="title">
+        Title:
+        <input type="text" name="title" id="title" />
+      </label>
+      { createCategories() }
+      <input type="submit" value="Submit" />
+    </form>
+  );
+}
+
+export default BooksForm;
```

Comment:
You forgot the following :
- it has a state with `title` and `category` 
- this file (`BookForm.js`), should be in `src/containers/` folder,  it's a "container" into which books will be filled , so relocate the fle 😊

Reply:
Oh that's right, BookForm will indeed become a container later on because it will communicate with the redux store and also manage its own state, is that the reason why it should be a container? My understanding was that if the components gets or sets state it is a container component. In other words, if it only generates HTML it is a presentational component.

Reply:
In regards to BookForm having its own state, I've definitely included that for Milestone 3, however despite the guidelines requiring this item in this Milestone, it isn't required in Milestone 2 on pathwright:
![image](https://user-images.githubusercontent.com/22228397/73593604-ff493f00-44d3-11ea-9be0-b2377ea93de7.png)

Reply:
oooh...concerning the file relocation, it could be anywhere..... just that it's on the requirements for it to be here (src/containers/ ), no deeper explanation to that 😊😊

![Screen Shot 2020-02-01 at 3 36 44 PM](https://user-images.githubusercontent.com/40712459/73593860-a9db5500-4508-11ea-9807-944b1390809d.png)